tensorflow/contrib/rnn/python/kernel_tests/core_rnn_test.py

```diff
@@ -307,6 +307,21 @@ def setUp(self):
     self._seed = 23489
     np.random.seed(self._seed)
 
+  def testDType(self):
+    # Test case for GitHub issue 16228
+    # Not passing dtype in constructor results in default float32
+    lstm = rnn_cell.LSTMCell(10)
+    input_tensor = array_ops.ones([10, 50])
+    lstm.build(input_tensor.get_shape())
+    self.assertEqual(lstm._bias.dtype, dtypes.float32_ref)
+
+    # Explicitly pass dtype in constructor
+    for dtype in ['float16', 'float32', 'float64']:
```

Comment:
for dtype in (dtypes.float16, dtypes.float32, dtypes.float64)

prefer objects to strings.

Reply:
Thanks @ebrevdo. The test has been updated.